l10n_be_mis_reports/data/mis_report_vat.xml

```diff
@@ -0,0 +1,472 @@
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">240</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_63">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_63</field>
+      <field name="description">Grid 63</field>
+      <field name="expression">crd[][('tax_line_id.tag_ids.name', 'like', '% 63')] </field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">250</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_xx">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_xx</field>
+      <field name="description">Grid xx (sum of 54, 55, 56, 57, 61, 63)</field>
+      <field name="expression">+grid_54+grid_55+grid_56+grid_57+grid_61+grid_63</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_4"/>
+      <field name="sequence">260</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_cadre_5">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">cadre_5</field>
+      <field name="description">Cadre V</field>
+      <field name="expression"></field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_2"/>
+      <field name="sequence">261</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_59">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_59</field>
+      <field name="description">Grid 59</field>
+      <field name="expression">deb[][('tax_line_id.tag_ids.name', 'like', '% 59')]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">270</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_62">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_62</field>
+      <field name="description">Grid 62</field>
+      <field name="expression">crd[][('tax_line_id.tag_ids.name', 'like', '% 62')]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">280</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_64">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_64</field>
+      <field name="description">Grid 64</field>
+      <field name="expression">deb[][('tax_line_id.tag_ids.name', 'like', '% 64')]</field>
```

Comment:
Now that version 3.1 is merged, it is possible to write

`deb[][('tax_line_id.tag_ids.id', '=', ref('l10n_be.tag_tag_64').id)]`

This would be more robust in case someone decides to translate or otherwise change the tag labels. 

Not blocking for me though.